db/migrate/20190116194130_cascade_image_revision_joins_on_revision_delete.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CascadeImageRevisionJoinsOnRevisionDelete < ActiveRecord::Migration[5.2]
+  def up
+    remove_foreign_key :versioned_revision_image_revisions,
+                       column: :revision_id
+
+    add_foreign_key :versioned_revision_image_revisions,
+                    :versioned_revisions,
+                    column: :revision_id,
+                    on_delete: :cascade
+  end
+
+  def down
+    remove_foreign_key :versioned_revision_image_revisions,
+                       column: :revision_id
+
+    add_foreign_key :versioned_revision_image_revisions,
```

Comment:
This looks like it's the same in both directions :-|.

Reply:
Crap, thanks